### ISSUE TITLE
How to use it? I try it but not working

### ISSUE THREAD

**Issue by zwl1619** [RavenEbonscale/Friday-Night-Funkin-bot#2]
How to use it? I try it but not working.
I can not search a result with `Raβen Smellyscally#432` in discord.

![image](https://user-images.githubusercontent.com/13177997/145258913-33c74a7f-3780-40eb-80aa-9bf77ee6a18d.png)

**Comment by RavenEbonscale** [comment]
Hmm mind sending me a link to where you download the game i can do a quick hotfix for it

**Comment by zwl1619** [comment]
@RavenEbonscale  It seems that all of mods based [FNF-PsychEngine](https://github.com/ShadowMario/FNF-PsychEngine) don't work.

**Comment by zwl1619** [comment]
I see. the window name need to be one of below names, right?
```
LPCWSTR w[6] =
	{ 
		L"Friday Night Funkin'", 
		L"Friday Night Funkin' Kade Engine",
		L"High Effort Ugh",L"Friday Night Funkin': Mid-Fight Masses - Selever Edition",
		L"Friday Night Funkin' Vs. Sasha",L"Friday Night Funkin' Modding Plus"
	};
```

**Comment by RavenEbonscale** [comment]
Yes,that is the issue I do have a python program that will give you the name of the correct windows
Edit: added the python program that should give you all the names of the programs you have open
